lib/Distribution/Sle/AgamaDevel.pm

```diff
@@ -10,35 +10,23 @@
 package Distribution::Sle::AgamaDevel;
 use strict;
 use warnings FATAL => 'all';
-use parent 'susedistribution';
+use parent Distribution::Opensuse::Leap::16Latest;
```

Comment:
shouldn't be this line after `package` line?

Reply:
only to avoid barewords, but I have added single quotes